### PR TITLE
added more cs tests

### PR DESCRIPTION
**Pull request by cfaur09**
## Reasoning
- 
- 
- 
  
## Proposed Changes
- 
- 
- 

## How to test
- 
- 
-

### CODE REVIEW

File: src/test/chain-simulator/collections.cs-e2e.ts [hunk 1]
```diff
@@ -149,4 +130,1185 @@ describe('Collections e2e tests with chain simulator', () => {
...
+
+      expect(response.status).toBe(200);
+      expect(response.data.length).toStrictEqual(1);
+
+      for (const transaction of response.data) {
+        expect(transaction.miniBlockHash).toStrictEqual(miniBlockHash[0]);
+      }
+    });
+
+    it('should return transactions for a collection of type NFT with a specific hashes', async () => {
```
Comment (bogdan-rosianu) [inline src/test/chain-simulator/collections.cs-e2e.ts:622]: with a specific hashes -> with specific hashes

File: src/test/chain-simulator/collections.cs-e2e.ts [hunk 2]
```diff
@@ -149,4 +130,1185 @@ describe('Collections e2e tests with chain simulator', () => {
...
+
+      expect(response.status).toBe(200);
+      expect(response.data.length).toStrictEqual(1);
+
+      for (const transfer of response.data) {
+        expect(transfer.miniBlockHash).toStrictEqual(miniBlockHash[0]);
+      }
+    });
+
+    it('should return transfers for a collection of type NFT with a specific hashes', async () => {
```
Comment (bogdan-rosianu) [inline src/test/chain-simulator/collections.cs-e2e.ts:1038]: with specific hashes

**Review by bogdan-rosianu — APPROVED** [review]

File: src/test/chain-simulator/accounts.cs-e2e.ts [hunk 3]
```diff
@@ -1545,4 +1528,461 @@ describe('Accounts e2e tests with chain simulator', () => {
       expect(response.data).toStrictEqual(expectedCount);
     });
   });
```
Comment (danielailie) [inline src/test/chain-simulator/accounts.cs-e2e.ts:1530]: I would split this into shorter files.. they are to big to even look at them

File: src/test/chain-simulator/collections.cs-e2e.ts [hunk 4]
```diff
@@ -149,4 +130,1185 @@ describe('Collections e2e tests with chain simulator', () => {
       expect(response.data.collection).toBe(collection[0]);
     });
   });
```
Comment (danielailie) [inline src/test/chain-simulator/collections.cs-e2e.ts:132]: I would split this into files based on the routes

**Review by danielailie — APPROVED** [review]

**Pull request merged**